### DISCUSSION TITLE
First-time contributions

### DISCUSSION THREAD

**Discussion by sebastiantia** [py-pdf/fpdf2#781] [Q&A] [answered]
Hello!

My name is Sebastian Tia and I'm currently studying Computer Engineering at the University of Waterloo. I am very interested in contributing to this project and would love a helping hand! A point to a beginner friendly first-issue would be fantastic and I'll take anything else I can get :)

**Comment by Lucas-C** [comment, marked as answer]
Hi and welcome @sebastiantia! 😊

I recently gave some pointers regarding newcomer-friendly issues there:
https://github.com/PyFPDF/fpdf2/issues/640#issuecomment-1508155357

Have you been using `fpdf2` in the past? If so, for what kind of task, out of curiosity?

**Reply by sebastiantia** [reply]
Thanks very much for the response! I have not used fpdf2 but found out about it when looking to generate invoices & reports dynamically.
